### ISSUE TITLE
Toggle Element for Payjoin send UX

### ISSUE THREAD

**Issue by benalleng** [MutinyWallet/mutiny-web#847]
Currently the payjoin flow in #608 only announces to the user that a particular send will be a payjoin and does not give them optionailty to disable it.  @DanGould has suggested we use a toggle to give the user to standardize the payjoin flow with what has been described in the bitcoin design guide for payjoin. As well @benthecarman has pointed out that the UI for the alert card is a bit awkward on the new Send screen.

> ![image](https://private-user-images.githubusercontent.com/15256660/298730649-30fed52b-21b2-48d8-94f7-266944597537.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5NzQ4ODUsIm5iZiI6MTcwNTk3NDU4NSwicGF0aCI6Ii8xNTI1NjY2MC8yOTg3MzA2NDktMzBmZWQ1MmItMjFiMi00OGQ4LTk0ZjctMjY2OTQ0NTk3NTM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIzVDAxNDk0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhZGUyMzU3MTEwYzhiNzQ5Y2I1MjNmZjZjMzc5NjEyY2RmNzUwMDQ3NDU4N2NiNTI1NDVlNDZkOGQ4YWZkODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dkwqbWzE9gscDnoDIOR8h2Cmtrc9CrcRcIMTILvz68Q) Tested and it worked for me. The payjoin notice is a bit big compared to everything else, not sure what a better way would be though

Moving forward I do think that we should think about this alert as being slightly different than the other error alerts that a user might encounter on the send screen. As those usually are an indication to the user that they should not/ cannot proceed.

And for reference, the bitcoin design guide uses the below UI

> Happy to follow up with a toggle that fits in the table like bitcoin design recommends
> 
> ![image](https://private-user-images.githubusercontent.com/8525467/279726504-4b841cf7-2884-4816-8984-76c6901c9c2a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5NzQ4NDYsIm5iZiI6MTcwNTk3NDU0NiwicGF0aCI6Ii84NTI1NDY3LzI3OTcyNjUwNC00Yjg0MWNmNy0yODg0LTQ4MTYtODk4NC03NmM2OTAxYzljMmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjNUMDE0OTA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjAzMjBmYmM3Zjk0OTc2YzRhMWZhODgxMWQyYzUxOWE0MDIzN2I3NmRkYTIxMzMyYTY3NGM0MTBhNDk5Yzg1ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.NKKJr8N9JFHAeGTJ3MIiHeprTUneP5H53KaORfXLF7s)

I do not think that this new alert should be a blocker for #608 as I think very few users will encounter this flow before some design decision can be made about this particular UI element

Lastly, we currently (to my knowledge), do not have any toggle/switch element and I know we have been trying to move away from our Kobalte dependency so instead of just mashing up something from their codebase I wanted to open up some design discussion about this element.

**Comment by TonyGiorgio** [comment]
I'm not sure what would be the point in disabling a privacy feature that both parties are capable of supporting. 

There's going to be many payment methods available soon (bolt12 too), and I feel like it'll eventually be too much.

**Comment by DanGould** [comment]
if the payment fails bc e.g the other party is offline you could set the toggle off to do a default on-chain spend. Or if the other party has pj fee requirements you don't agree to

**Comment by DanGould** [comment]
@benalleng Can't you use [`<input type="checkbox">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox) toggle?

**Comment by benalleng** [comment]
Yeah didn't even think about the checkbox, ben reminded me that we already use that in the backup flow. Likely will go with that

**Comment by TonyGiorgio** [comment]
The payment pages have changed drastically and will continue to change. I think it's fine to leave it for now or default to the on chain method after it fails first instead of implementing check boxes.
